### PR TITLE
Improve Font Page README.md

### PR DESCRIPTION
**Pull request by jgehrig**
The landing page could use improvement.

We should have a screenshot.

Several questions are asked repeatedly by users. We should make the questions/answers available in an obvious place: `GuiFont!`, `GuiTabline`, `GuiPopupmenu`, etc.

It would be nice to have a space to clearly describe common configuration options an features. Visuals would be useful.

### CODE REVIEW

**Comment by jgehrig** [comment]
@jdhao

I've seen you file some bug reports on Windows for changes only available in `master`. Do you have a simple way of installing `neovim-qt` nighties on Windows?

If so, I would like to add this to the improved README. Thanks!

**Comment by jdhao** [comment]
@jgehrig  I think the master build was downloaded from the CI. There is no automatic way to do this now. Maybe you can add a nightly build release for nvim-qt, just like what neovim currently does? Then it should be easy to install nvim-qt nightly version.

**Comment by jgehrig** [comment]
@jdhao

Thanks for the confirmation, I do the same.

Getting automatic nightly releases from the CI is on my TODO list :)

File: README.md [hunk 1]
```diff
@@ -1,99 +1,224 @@
...
+
+### Linux
+
+Neovim Qt is packaged for many distributions, use your favorite package manager.
+
+If your distribution is not listed here, please create an Issue with instructions.
+
+#### ArchLinux
+
+NeovimQt is available from this [AUR Package](https://archlinux.org/packages/community/x86_64/neovim-qt/).
```
Comment (fuelen) [inline README.md:63]: This is not AUR package.

Comment (jgehrig) [reply README.md:63]: Sorry, I don't run Arch...

Do you have better instructions/links you can provide? Thanks!

Comment (fuelen) [reply README.md:63]: `NeovimQt is available from this [AUR Package]` -> `NeovimQt is available in [community repository]`

**Pull request merged**